keps/sig-node/20190226-pod-overhead.md

```diff
@@ -82,7 +106,7 @@ introduced which will update the `Overhead` field in the workload's `PodSpec` to
 what is provided for the selected RuntimeClass, if one is specified.
 
 Kubelet's creation of the pod cgroup will be calculated as the sum of container
-`ResourceRequirements` fields, plus the Overhead associated with the pod.
+`ResourceRequirements.Limits` fields, plus the Overhead associated with the pod.
 
 The scheduler, resource quota handling, and Kubelet's pod cgroup creation and eviction handling
```

Comment:
TODO: currently not clear how resource quota handling will take Overhead into account.  Probably makes sense to make this optional (default to subtract the overhead if applicable  to better match the current status quo), and leave it to the administrator to decide. 

Similarly, early RFC discussions included augmenting CRI interface.  This should be added:
```
I suggest adding the LinuxContainerResources message to the LinuxPodSandboxConfig,
as an optional field. Unlike the Resources field on the Kubernetes PodSpec, the resources
field on the LinuxPodSandboxConfig matches the pod-level limits (i.e. the total of pod &
container limits). The field is only provided when a pod-level limit is enforced.
```

Reply:
For resource quota, I think @derekwaynecarr or @bsalamat might be able to provide some guidance.

Just my 2c - I think a configurable option makes sense, but I still think we should treat the decision of the default behaviour as if there wasn't an option (because in practice very few will change it). I think accounting the overhead to the user makes sense as a default.

Reply:
i prefer to charge the user for the overhead by default.

Reply:
+1 to @derekwaynecarr's point

Reply:
Updated accordinly. PTAL.